sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collationExpressions.scala

```diff
@@ -38,7 +38,7 @@ import org.apache.spark.sql.types._
     Examples:
       > SET spark.sql.collation.enabled=true;
       spark.sql.collation.enabled	true
-      > SELECT COLLATION('Spark SQL' _FUNC_ UTF8_LCASE);
+      > SELECT _FUNC_('Spark SQL', 'UTF8_LCASE');
       UTF8_LCASE
```

Comment:
the output of the `collate` function will be the first argument (`Spark SQL`) and not the given collation

Reply:
Thank you.